tmva/sofie/src/RModel.cxx

```diff
@@ -153,6 +168,11 @@ bool RModel::IsInitializedTensor(const std::string& tensorName) const {
     return fInitializedTensors.find(name) != fInitializedTensors.end();
 }
 
+bool RModel::IsDynamicTensor(const std::string& tensorName) const {
+   std::string name = UTILITY::Clean_name(tensorName);
```

Comment:
The name for the namespace UTILITY seems a bit inappropriate. Can this be replaced with something like "ROOT::Internal::Sofie::Utility"? Otherwise at the prompt one can stumble on the namespace UTILITY for example...

Reply:
(Like RDF)

Reply:
UTILITY is an internal namespace under TMVA::Experimental::SOFIE